exec/csnex/Exec.cs

```diff
@@ -43,7 +43,9 @@ public int Run(bool EnableAssertions)
 #region PUSHx Opcodes
         void PUSHB()
         {
-            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+            bool val = bytecode.code[ip + 1] != 0;
```

Comment:
`bool` is used here and `Boolean` is used in other places. I think they're aliases for the same type in C#, but try to be consistent.

Reply:
Fixed.